test/change_fee_receiver.test.ts

```diff
@@ -0,0 +1,137 @@
+import { ONE_ALPH, web3 } from '@alephium/web3'
+import { getSigner } from '@alephium/web3-test'
+import { PrivateKeyWallet } from '@alephium/web3-wallet'
+import { getPool, expectError } from '../src/testUtils'
+import { AddFeeTier, ChangeFeeReceiver, CreatePool } from '../artifacts/ts'
+import { deployInvariant, deployTokenFaucet, MAP_ENTRY_DEPOSIT } from '../src/utils'
+
+web3.setCurrentNodeProvider('http://127.0.0.1:22973')
+let sender: PrivateKeyWallet
+
+describe('change fee receiver tests', () => {
+  beforeAll(async () => {
+    sender = await getSigner(ONE_ALPH * 1000n, 0)
+  })
+
+  test('should change the fee receiver', async () => {
+    const protocolFee = 10n ** 10n
+    const fee = 6n * 10n ** 9n
+    const tickSpacing = 10n
+
+    const invariant = await deployInvariant(sender, protocolFee)
+
+    const amount = 1000000n + 1000n
+
+    const token0 = await deployTokenFaucet(sender, '', '', amount, amount)
+    const token1 = await deployTokenFaucet(sender, '', '', amount, amount)
```

Comment:
we have a function `initTokensXY` for this in `testUtils.ts`, additionally it returns sorted tokens (we refer to them as tokenX and tokenY in that case)